plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/adapter/controllers/CDK3LaunchController.java

```diff
@@ -107,10 +107,13 @@ protected void performOverrides(ILaunchConfigurationWorkingCopy workingCopy) thr
 
 		Map<String, String> env = workingCopy.getAttribute(ENVIRONMENT_VARS_KEY, (Map<String, String>) null);
 		env = (env == null ? new HashMap<>() : new HashMap<>(env));
-		String msHome = s.getAttribute(CDK3Server.MINISHIFT_HOME, (String) null);
-		if (msHome != null) {
-			env.put("MINISHIFT_HOME", msHome);
-		}
+		
+		String home = System.getProperty("user.home");
+		String defaultMinishiftHome = new File(home, ".minishift").getAbsolutePath();
```

Comment:
it would help if ".minishift" was a constant, would be prominent for latter reviewers